pkg/addons/data/ConfigMap/openshift-azure-monitoring/prom-mdm-converter-rules.yaml

```diff
@@ -0,0 +1,23 @@
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: prom-mdm-converter-rules
+  namespace: openshift-azure-monitoring
+data:
+  rules.yaml: |
+    defaultMetricAllow: false
+    defaultDimensionAllow: true
+    metrics:
+      kube_node_info:
```

Comment:
@sozercan can we make this "Forward all" in any way? So we would control those on our Prometheus level so we can limit how much work it needs to do.